mhcflurry/class1_affinity_prediction/train_allele_specific_models_command.py

```diff
@@ -8,8 +8,8 @@
 
 import pandas
 
-from .class1_affinity_predictor import Class1AffinityPredictor
-from ..common import configure_logging
+from mhcflurry import Class1AffinityPredictor, downloads
```

Comment:
Files in the mhcflurry package (like this one) should import mhcflurry functionality using local imports (like `from . import ...`) not from `mhcflurry` (this is just a convention but it's a common one)